deployment/ansible/deploy.yml

```diff
@@ -5,7 +5,7 @@
 
   vars:
     project_root: "{{ ansible_env.HOME }}/node"
-    genesis_accounts_path: "{{ project_root }}/lib/aecore-0.1.0/priv/.genesis/accounts.json"
+    genesis_accounts_path: "{{ project_root }}/data/aecore/.genesis/accounts.json"
```

Comment:
Once this is deployed in our environments, devops may delete the old file for avoiding confusion. Cc @dincho

Reply:
Following PT task would be nice to not loose the track

Reply:
@dincho https://www.pivotaltracker.com/story/show/153441285